src/implementations/twig/components/tag/tag-set.html.twig

```diff
@@ -44,6 +46,7 @@
       <li class="ecl-tag-set__item">
         {% include '@ecl/tag/tag.html.twig' with _item|merge({
           icon_path: _item.icon_path|default(_icon_path),
+          color_mode: _color_mode,
```

Comment:
mmh...don't you foresee the fantasy of an editor introducing a tag list with each tag using a different color mode..? :)